misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,40 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  val DEFAULT_BACKOFF = ExponentialBackoff(
+      baseDelay = Duration.ofMillis(10),
+      maxDelay = Duration.ofMillis(500),
+      jitter = Duration.ofMillis(5)
+  )
+
+  /**
+   * Retries any [AmazonDynamoDBException] or [PleaseRetryException]. [DontRetryException] will
+   * not be retried.
+   */
+  fun <T> dynamoDbRetry(
+    retries: Int = 3,
```

Comment:
why 3?

Reply:
That's the default max in the hibernate transacter

Reply:
sounds reasonable